sdk/core/azure-core-http-jdk-httpclient/src/main/java/com/azure/core/http/jdk/httpclient/implementation/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing implementation classes used by the Azure Core HTTP client.
```

Comment:
Maybe we want to be a bit more specific?
```suggestion
 * Package containing implementation classes used by the JDK-backed Azure Core HTTP client.
```